### PR TITLE
hypixel bypasses

### PR DESCRIPTION
**Pull request by liquidsquid1**
- hypixel velocity fix
- hypixel lowhop
- hypixel target strafe

### CODE REVIEW

**Comment by localpthebest** [comment]
the way to get the nuker issues solved is by simply making the worst code ever known to man and making a pr then izuna will be forced to remake it but good

**Comment by liquidsquid1** [comment]
> the way to get the nuker issues solved is by simply making the worst code ever known to man and making a pr then izuna will be forced to remake it but good

Noted. /j

**Comment by eonianisnoob** [comment]
liquidskid bypass

**Comment by liquidsquid1** [comment]
> liquidskid bypass

I don't skid.

**Comment by localpthebest** [comment]
> > liquidskid bypass
> 
> I don't skid.

nightx would say otherwise

**Comment by ghost** [comment]
skidded from sigma hypixel funcraft fly

**Comment by liquidsquid1** [comment]
> skidded from sigma hypixel funcraft fly

it just isnt.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityWatchdog.kt [hunk 1]
```diff
@@ -75,12 +54,7 @@ internal object VelocityWatchdog : Choice("Watchdog") {
     @Suppress("unused")
     private val gameHandler = repeatable {
         if (player.isOnGround) {
-            blinkUntilGround = false
-        }
-
-        // Check if there is enemy nearby
-        if (hits > 0 && world.findEnemy(range = 0f..4f) == null) {
-            hits = 0
+            absorbedVelocity = false;
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityWatchdog.kt]: remove ;

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityWatchdog.kt [hunk 2]
```diff
@@ -20,37 +20,18 @@ package net.ccbluex.liquidbounce.features.module.modules.combat.velocity.mode
...
-
-    /**
-     * Blink until we reach ground after cancelling in air
-     */
-    private var blinkUntilGroundOpt by boolean("BlinkUntilGround", false)
-    private var blinkUntilGround = false
-
-    internal val shouldLag: Boolean
-        get() = isActive && super.handleEvents() && pause == 0 && blinkUntilGroundOpt && blinkUntilGround
+    private var absorbedVelocity = false;
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityWatchdog.kt]: remove ;

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleTargetStrafe.kt [hunk 3]
```diff
@@ -170,11 +173,36 @@ object ModuleTargetStrafe : Module("TargetStrafe", Category.MOVEMENT) {
...
 
             // Perform the strafing movement
-            event.movement.strafe(
-                yaw = toDegrees(atan2(-strafeVec.x, strafeVec.z)).toFloat(),
-                speed = player.sqrtSpeed,
-                keyboardCheck = false
-            )
+
+            if (hypixel && ModuleSpeed.enabled) {
+
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleTargetStrafe.kt]: remove space

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/watchdog/SpeedHypixelLowHop.kt [hunk 4]
```diff
@@ -27,65 +27,71 @@ import net.ccbluex.liquidbounce.event.events.PlayerJumpEvent
...
+        var shouldStrafe = false
     }
 
-    private var wasFlagged = false
+    private var glide by boolean("Glide", true)
+
     private var airTicks = 0
 
     val repeatable = repeatable {
+
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/watchdog/SpeedHypixelLowHop.kt]: remove space

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/watchdog/SpeedHypixelLowHop.kt [hunk 5]
```diff
@@ -27,65 +27,71 @@ import net.ccbluex.liquidbounce.event.events.PlayerJumpEvent
...
-
+        var shouldStrafe = false
     }
 
-    private var wasFlagged = false
+    private var glide by boolean("Glide", true)
+
     private var airTicks = 0
 
     val repeatable = repeatable {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/watchdog/SpeedHypixelLowHop.kt:50]: @Supress("unusued")
private val repeatable = ...

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityWatchdog.kt (hunk 1 above)
Comment (liquidsquid1) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityWatchdog.kt]: this is a really annoying habit, especially that its grey in intellij so i can't see :sob:

**Comment by liquidsquid1** [comment]
Putting this in draft as I want to have a go at making a 7 tick lowhop.

**Pull request merged**